### PR TITLE
Added pip, pipenv and related dev packages

### PR DESCRIPTION
**Pull request by viktigpetterr**

### CODE REVIEW

**Comment by sweoggy** [comment]
Please combine into a single RUN statement

**Comment by viktigpetterr** [comment]
@sweoggy Sure thing

File: Dockerfile [hunk 1]
```diff
@@ -117,7 +117,8 @@ RUN cd / \
 
 ENV GRADLE_HOME /opt/gradle/gradle-${GRADLE_VERSION}
 ENV PATH ${GRADLE_HOME}/bin:${PATH}
-
+# Install pip and pipdeptree
+RUN apt install python3-pip && pip install pipdeptree
```
Comment (sweoggy) [inline Dockerfile]:
```suggestion
RUN apt install python3-pip && pip install pipdeptree -y
```

Or build fails

**Comment by sweoggy** [comment]
Build still fails unfortunately. Try to make the build succeed locally first and fix any error which occurs :)

File: Dockerfile [hunk 2]
```diff
@@ -117,7 +117,8 @@ RUN cd / \
 
 ENV GRADLE_HOME /opt/gradle/gradle-${GRADLE_VERSION}
 ENV PATH ${GRADLE_HOME}/bin:${PATH}
-
+# Install python and pip and related dev packages. 
+RUN apt install python3 python3-dev python2-dev python2 python3-pip libffi-dev libssl1.0-dev && pip3 install pipenv
```
Comment (sweoggy) [inline Dockerfile]:
```suggestion
RUN apt install python3 python3-dev python3-pip libffi-dev libssl1.0-dev && pip3 install pipenv
```

**Pull request closed without merging**